bootloaders/riotboot/README.md

```diff
@@ -60,6 +60,10 @@ A board capable to use riotboot must meet the following requirements:
 The above requirements are usually met if the board succeeds to execute
 the riotboot test in tests/.
 
+When building the bootloader, the global define `RIOTBOOT` is available. You
```

Comment:
Isn't this also the case for `MODULE_RIOTBOOT`?

Reply:
Yes, but MODULE_RIOTBOOT is also available when *not* building the bootloader but using the riotboot module.

Reply:
I have updated the docs to clarify the difference between `MODULE_RIOTBOOT` and `RIOTBOOT`.